library/globals.inc.php

```diff
@@ -2968,6 +2968,14 @@
       '0',                               // default
       xl('Show demo system "Save/Load database presets" menu')
     ),
+    
+    'report_itemizing_pqrs' => array(
+      xl('Enable MIPS report itemization'),		// for itemizing reports
+      'text',                           // data type
+      '1',                     // default
+      xl('Creates patient lists from reports')
+    ),
+    
 
     'pqrs_creator' => array(
       xl('MIPS Report Creator Name'),		// for XML generation
```

Comment:
No, pqrs_creator is a registry name (well documented stuff for those doing QA reporting).  This is subject to change once we get the 2017 standards published by CMS....any day now...I am sure..